{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2020-35579"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "tindy2013 subconverter 0.6.4 has a /sub?target=%TARGET%\u0026url=%URL%\u0026config=%CONFIG% API endpoint that accepts an arbitrary %URL% value and launches a GET request for it, but does not consider that the external request target may indirectly redirect back to this original /sub endpoint. Thus, a request loop and a denial of service may occur."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/tindy2013/subconverter/issues/284",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/tindy2013/subconverter/issues/284"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-12-20T07:15Z",
  "publishedDate": "2020-12-20T07:15Z"
}